js-extensions/src/init/blueocean-core-js.js

```diff
@@ -0,0 +1,22 @@
+import * as modules from '@jenkins-cd/js-modules';
+
+/**
+ * Executes extension init and hack for blueocean-web i18n
+ */
+export function execute(done, config) {
+	// Get the extension list metadata from Jenkins.
+	// have a special handling for core-js, must load the web bundle...
+	const Extensions = require('@jenkins-cd/js-extensions');
+	const appRoot = document.getElementsByTagName('head')[0].getAttribute('data-appurl');
+	
+	Extensions.init({
+	    extensionData: window.$blueocean.jsExtensions,
+	    classMetadataProvider: (type, cb) => {
+	        const fetch = require('../../../../../src/js/fetch').Fetch;
+	        fetch.fetchJSON(`${appRoot}/rest/classes/${type}/`).then(cb).catch(fetch.consoleError);
+	    },
+	});
+	
+	// core-js has references to blueocean-web translations
+	require('../../../../../src/js/i18n/bundle-startup').execute(done, { hpiPluginId: 'blueocean-web' });
```

Comment:
Would it be any better to shift the translation files to blueocean-core-js instead? Since we're building it as a HPI now that would actually be feasible?

Reply:
big +1 We depended on `web` because we had to, now we do not anymore and we can make a clean cut.

Reply:
@cliffmeyers @scherler yes, it would be best. However, keep in mind I'm trying to minimize changes with this PR to the minimum viable for making the split happen. After which time, we don't have to keep doing the publish dance, it will be much faster to move things around like this. In this particular case, it would involve updating `js-extensions`, but that's a "one-time" thing instead of an "every-time" thing with JDL and core-js...